src/Bridge/Doctrine/Persister/ManagerRegistryPersister.php

```diff
@@ -42,12 +42,12 @@ public function __construct(ManagerRegistry $registry)
         $managers = $registry->getManagers();
 
         foreach ($managers as $manager) {
-            $this->persisters[get_class($manager)] = new ObjectManagerPersister($manager);
+            $this->persisters[spl_object_hash($manager)] = new ObjectManagerPersister($manager);
```

Comment:
This allows having multiple instance of `Doctrine\ORM\EntityManager`

Reply:
👍